src/ext/java/org/opentripplanner/ext/ridehailing/RideHailingServiceParameters.java

```diff
@@ -0,0 +1,13 @@
+package org.opentripplanner.ext.ridehailing;
+
+/**
+ * Interface and concrete configurations for ride hailing services.
+ */
+public sealed interface RideHailingServiceParameters {
```

Comment:
I think I see what you are trying to do here, but remove the interface - It is going to break if you want to include more than one provider anyway.

Reply:
Done in 02e09a694b82b68351b5b87d3ff92823776074b1